src/tests/ftest/scripts/main.sh

```diff
@@ -223,7 +223,7 @@ fi
 
 launch_args="-jcrisa"
 # can only process cores on EL7 currently
-if [ "$(lsb_release -s -i)" = "CentOS" ]; then
+if [ "$(lsb_release -s -i)" = "CentOS" ] || [ "$(lsb_release -s -i)" = "openSUSE" ]; then
```

Comment:
(style)  line over 80 characters

Reply:
Since you have to fix the linting issue anyway, it's better to only execute the command once and store it's value and then reference that:
```suggestion
name="$(lsb_release -s -i)"
if [ "$name" = "CentOS" ] || [ "$name" = "openSUSE" ]; then
```

Reply:
Applied, lint issue should be marked as fixed.